Dockerfile.repo

```diff
@@ -1,23 +1,27 @@
-FROM centos:centos7
+FROM quay.io/centos/centos:stream8
 
 # R, Java
 ENV JAVA_VERSION 1.8.0
 ENV JAVA_HOME /usr/lib/jvm/java
-ENV LD_LIBRARY_PATH $JAVA_HOME/jre/lib/amd64:$JAVA_HOME/jre/lib/amd64/server/
-RUN echo "$JAVA_HOME/jre/lib/amd64/server/" > /etc/ld.so.conf.d/rApache_rJava.conf && \
+RUN echo "$JAVA_HOME/jre/lib/$(arch)/server/" > /etc/ld.so.conf.d/rApache_rJava.conf && \
+    export LD_LIBRARY_PATH=$JAVA_HOME/jre/lib/$(arch):$JAVA_HOME/jre/lib/$(arch)/server/ && \
     /sbin/ldconfig
 
 RUN \
   useradd -u 1001 -ms /bin/bash builder && \
-  yum update -y && \
-  yum upgrade -y && \
-  yum install -y epel-release && \
-  yum install -y rpm-build which make wget tar httpd-devel libapreq2-devel libcurl-devel protobuf-devel openssl-devel libpng-devel java-1.8.0-openjdk java-1.8.0-openjdk-devel && \
-  yum-builddep -y R-devel && \
-  yum clean all
-
-ENV R_VERSION 3.6.3
-RUN curl -O https://cran.rstudio.com/src/base/R-3/R-${R_VERSION}.tar.gz && \
+  echo "fastestmirror=1" >> /etc/dnf/dnf.conf && \
+  dnf update -y && \
+  dnf upgrade -y && \
+  dnf install -y epel-release && \
+  dnf config-manager --set-enabled powertools && \
+  dnf module enable -y postgresql:13 && \
+  dnf install postgresql-server-devel postgresql-devel -y && \
+  dnf install -y rpm-build which make wget tar httpd-devel libapreq2-devel libcurl-devel protobuf-devel openssl-devel libpng-devel java-1.8.0-openjdk java-1.8.0-openjdk-devel curl-devel libxml2-devel llvm5.0-devel llvm-toolset sudo cmake initscripts && \
+  dnf builddep -y R-devel && \
+  dnf clean all
+
+ENV R_VERSION 4.1.2
+RUN curl -O https://cran.rstudio.com/src/base/R-4/R-${R_VERSION}.tar.gz && \
```

Comment:
We changed R versions because the `json` R package is > 4 dependent now and my switch to centos stream8 for multi-arch builds installs latest packages (not easy to lock in R).